orca-queue-tck/src/main/kotlin/com/netflix/spinnaker/orca/q/QueueIntegrationTest.kt

```diff
@@ -309,10 +309,16 @@ abstract class QueueIntegrationTest {
         requisiteStageRefIds = listOf("2a1")
       }
       stage {
-        refId = "2b"
-        type = "dummy"
+        refId = "2b1"
+        type = "wait"
+        context = mapOf("waitTime" to 2)
         requisiteStageRefIds = listOf("1")
       }
+      stage {
+        refId = "2b2"
+        type = "dummy"
+        requisiteStageRefIds = listOf("2b1")
+      }
```

Comment:
`keiko-sql` does not attempt to guarantee FIFO semantics; messages should only be delivered at or after their ready time and only once (within the `ackTimeout`), however ready messages are not necessarily read in insertion order. `RedisQueue` doesn't strictly guarantee ordered message executions either when running across multiple instances but does when running on a single instance such as during the execution of these tests.

Test changes here are made for tests with parallel execution branches, where assumptions were made that parallel branches without explicit ordering would still execute in a determinate order.

For this test, the original expectation:
```
       2a1 (TERMINAL) -> 2a2 (NOT_STARTED) ------\
1  <                                              3 (NOT_STARTED)
      2b (SUCCEEDED)  ----------------------------/
```
becomes:
```
       2a1 (TERMINAL) -> 2a2 (NOT_STARTED) -----------------\
1  <                                                         3 (NOT_STARTED)
      2b1 (SUCCEEDED or CANCELED) -> 2b2 (NOT_STARTED)  ----/
```
because while on most executions, execution of the A and B branches will be interleaved in exact message insertion order such that 2b/2b1 processes its `RunTask` message before 2a1 persists the execution with `isCanceled` set to true, it become possible for the A branch's `StartStage` and subsequent `RunTask` to complete before the B branch's `RunTask` is processed, so it can no longer be assumed to always `SUCCEED`. That should be ok. In a real HA Orca deployment using keiko-redis, the same thing could happen depending on the timing of GC pauses, network blips, or instance speed.